.editorconfig

```diff
@@ -8,7 +8,9 @@ trim_trailing_whitespace = true
 insert_final_newline = true
 
 [{package.json,.*rc,*.yml}]
-indent_style = space
+indent_size = 2
```

Comment:
I know it isn't from this PR's work, but looks like we have a duplicated setting here & on line 14. :thinking: